theme/admin.py

```diff
@@ -16,9 +18,32 @@ class HomePageAdmin(PageAdmin):
     inlines = (IconBoxInline,)
 
 
+class UserQuotaForm(forms.ModelForm):
+    user = forms.ModelChoiceField(queryset=User.objects.exclude(is_superuser=True))
+
+    class Meta:
+        model = UserQuota
+        fields = ['user', 'allocated_value', 'used_value', 'unit', 'zone']
+
+    def save(self, *args, **kwargs):
+        instance = super(UserQuotaForm, self).save(commit=False)
+        instance.user = self.cleaned_data['user']
+        return instance
+
+
 class QuotaAdmin(admin.ModelAdmin):
+    model = UserQuota
+
     list_display = ('user', 'allocated_value', 'used_value', 'unit', 'zone')
-    list_filter = ('zone',)
+    list_filter = ('zone', 'user__username', )
+
+    def get_form(self, request, obj=None, **kwargs):
+        # use a customized form class when adding a UserQuota object so that
+        # the foreign key user field is available for selection.
+        if obj is None:
+            return UserQuotaForm
+        else:
+            return super(QuotaAdmin, self).get_form(request, obj, **kwargs)
```

Comment:
@pkdash I added comment above trying to clarify the logic here. The customized ```UserQuotaForm``` is only used when the input parameter ```obj``` is None which means "Add UserQuota" button is clicked from admin interface which adds a new model instance (hence the pass-in ```obj``` is None), in which case, a customized form has to be used to have the foreign key User field available for selection. Otherwise, when obj is not None, which means an existing UserQuota instance is clicked for modification from the admin interface, in which case the default Django-provided form can be used since the user foreign key is fixed to that user and does not have to show up in the form, but instead other fields can be changed for this UserQuota instance for a specific User. Hope this clarify things enough, but let me know if you have further questions.